cmake/shared/CodeAnalysis.cmake

```diff
@@ -0,0 +1,24 @@
+add_custom_target(analyse
+  COMMENT "Performing code analysis..."
+  )
+
+add_custom_target(analyse-mlint
+  COMMENT "- Performing MATLAB analysis (Mlint)..."
+  BYPRODUCTS "${CMAKE_CURRENT_BINARY_DIR}/mlint.out"
+  COMMAND matlab -nodisplay <<< "\"addpath('${Herbert_ROOT}/../admin');lint_json({'${CMAKE_SOURCE_DIR}/**/*.m'},'${CMAKE_CURRENT_BINARY_DIR}/mlint.json');\""
```

Comment:
The CmakeList.txt in _test folder include code managing the MATLAB path before executing tests